src/environments/environment.ts

```diff
@@ -32,6 +32,9 @@ export const environment = {
   defaultLanguage: window['env']['defaultLanguage'] || 'en-US',
   supportedLanguages: window['env']['supportedLanguages'] || 'en-US,fr-FR',
   headOfficeID: window['env']['headOfficeID'] || '1',
+  //Matomo instance config
+  matomoSiteId: window['env']['matomoSiteId'] || 1,
+  matomoSiteUrl: window['env']['matomoSiteUrl'] || 'https://analytics.test.oneacrefund.org',
```

Comment:
I would prefer if these were deployment-time env vars... Does this web app not support deploy-time configuration, or does it have to be compile-time?

Reply:
Not really sure if we support deploy-time for the web apps, @shejasabin @amydoxym  any ideas.